components/support/nimbus-fml/src/workflows.rs

```diff
@@ -158,6 +161,63 @@ pub(crate) fn fetch_file(files: &LoaderConfig, nm: &str) -> Result<()> {
     Ok(())
 }
 
+pub(crate) fn validate(cmd: &ValidateCmd) -> Result<()> {
+    let files: FileLoader = TryFrom::try_from(&cmd.loader)?;
+    let cwd = cmd.loader.cwd.to_str().unwrap();
+
+    let filename = &cmd.manifest;
+    let file_path = files.file_path(filename)?;
+    let parser: Parser = Parser::new(files, file_path.clone())?;
+    let mut loading = HashSet::new();
+    let manifest_front_end = parser.load_manifest(&file_path, &mut loading)?;
+
+    println!(
+        "Loaded modules:\n- {}\n",
+        loading
+            .iter()
+            .map(|id| id.to_string().replace(cwd, ""))
+            .collect::<Vec<String>>()
+            .join("\n- ")
+    );
+
+    println!("Validation manifest for different channels:");
```

Comment:
If there are no channels, the manifest isn't necessarily invalid, it may be incomplete or be involved as an include. 

We should likely add a message, otherwise the validate command looks broken.